pkgs/applications/editors/vim/plugins/nvim-treesitter/overrides.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, callPackage, tree-sitter, nodejs }:
+
+self: super:
+
+let
+  builtGrammars = callPackage ./generated.nix {
+    buildGrammar = callPackage ../../../../../development/tools/parsing/tree-sitter/grammar.nix { };
+  };
+
+  allGrammars = lib.filter lib.isDerivation (lib.attrValues builtGrammars);
+
+  # Usage:
+  # pkgs.vimPlugins.nvim-treesitter.withPlugins (p: [ p.c p.java ... ])
+  # or for all grammars:
+  # pkgs.vimPlugins.nvim-treesitter.withAllGrammars
```

Comment:
`withPlugins` and `withAllGrammars` use two different term "plugin" and "grammar", which is inconsistent.

Reply:
I'm not sure what I should do here, tree-sitter is also using similarly inconsistent wordings